src/redux/books/books.js

```diff
@@ -1,46 +1,90 @@
-import { v4 as uuidv4 } from 'uuid';
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/BbK8z59qXzWwp214YswA/books';
 
 const ADD_BOOK = 'bookstore/books/ADD_BOOK';
-const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
-
-const initialState = [
-  {
-    id: uuidv4(),
-    title: 'The Successful Women',
-    category: 'Sci-Fi',
-    author: 'Fatima Zahir',
-  },
-  {
-    id: uuidv4(),
-    title: 'Hayya My Life',
-    category: 'Motivational',
-    author: 'Fatimaaa',
-  },
-  {
-    id: uuidv4(),
-    title: 'The Life & Death',
-    category: 'Sci-Fi',
-    author: 'John Adatya',
-  },
-  {
-    id: uuidv4(),
-    title: 'The Wealth or The Love',
-    category: 'Sci-Fi',
-    author: 'Abraham Joliey',
-  },
-];
+// const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
```

Comment:
- [optional] Commented code makes your project look messy. Avoid committing them. If you need to review a previous version of your code, you can always use git history.